interop-cats-laws/src/test/scala/scalaz/zio/interop/catzSpec.scala

```diff
@@ -29,41 +25,21 @@ class catzSpec
     with GenIO
     with RTS {
 
-  /**
-   * Silences `System.err`, only printing the output in case exceptions are
-   * thrown by the executed `thunk`.
-   */
-  def silenceSystemErr[A](thunk: => A): A = synchronized {
-    // Silencing System.err
-    val oldErr    = System.err
-    val outStream = new ByteArrayOutputStream()
-    val fakeErr   = new PrintStream(outStream)
-    System.setErr(fakeErr)
-    try {
-      val result = thunk
-      System.setErr(oldErr)
-      result
-    } catch {
-      case NonFatal(e) =>
-        System.setErr(oldErr)
-        // In case of errors, print whatever was caught
-        fakeErr.close()
-        val out = outStream.toString("utf-8")
-        if (out.nonEmpty) oldErr.println(out)
-        throw e
-    }
-  }
+  override def defaultHandler: ExitResult.Cause[Any] => IO[Nothing, Unit] = _ => IO.unit
 
   def checkAllAsync(name: String, f: TestContext => Laws#RuleSet): Unit = {
     val context = TestContext()
     val ruleSet = f(context)
 
     for ((id, prop) ← ruleSet.all.properties)
       test(name + "." + id) {
-        silenceSystemErr(check(prop))
+        check(prop)
       }
   }
 
+  (1 to 50).foreach { s =>
```

Comment:
We don't need to run those 50 times anymore, do we?

Reply:
Hopefully we don't. But, these tests are cheap enough to run and each iteration gives a slightly higher chance of spotting a regression. I know this is a defensive code smell, but IMHO justified by the domain - these are effectively integration tests, that also highly depend on JVM behavior and on upstream cats-effect.